sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Common/RuleTest.cs

```diff
@@ -130,6 +130,7 @@ public void AllParameterizedRules_AreDisabledByDefault()
                 .Where(RuleFinder.IsParameterized)
                 .Select(type => (DiagnosticAnalyzer)Activator.CreateInstance(type))
                 .SelectMany(analyzer => analyzer.SupportedDiagnostics)
+                .Where(analyzer => !IsSecurityHotspot(analyzer))
```

Comment:
`AllParameterizedRules_AreDisabledByDefault` - I am confused - does `Parameterized` refer to `WithNotConfigurable`?

Reply:
No, here `Parameterized` refer to a rule that has a `RuleParameter` aatribute, so that has a SonarQube parameter